tests/data/googletest/fail/record_test_result.json

```diff
@@ -15,7 +15,7 @@
       "duration": 0.001,
       "status": 0,
       "stdout": "",
-      "stderr": "github.com/launchableinc/rocket-car-googletest/test_a.cpp:21\nExpected equality of these values:\n  false\n  true\ngithub.com/launchableinc/rocket-car-googletest/test_a.cpp:21\nExpected equality of these values:\n  false\n  true",
```

Comment:
`github.com/launchableinc/rocket-car-googletest/test_a.cpp:21\nExpected equality of these values:\n  false` has been included in the modified text.